book_source/03_topical_pages/93_installation/03_install_OS/04_Installing-PEcAn-OSX.Rmd

```diff
@@ -86,7 +154,20 @@ make
 sudo make install
 ```
 
-#### Apache Configuration
+#### Install DuckDB
+
+##### Option 1: Install using homebrew
+
+```bash
+brew install duckdb
```

Comment:
Looks to me like the {duckdb} R package includes everything with no need to install any libraries separately. https://duckdb.org/docs/installation/?version=stable&environment=r